### PR TITLE
Bump RC to version 0.1.6

### PR DESCRIPTION
**Pull request by bleehu**
https://areyoudownwithotpyesyouknowme.tumblr.com/post/48641430196

### CODE REVIEW

**Comment by Turtlelord26** [comment]
Can we revert that RC to dev merge? It's erased some recent work and resurrected a few things that should stay gone.

**Comment by Turtlelord26** [comment]
Formal review is ready to go, just waiting on the 561f3ef reversion to see what changes.

The only real problem I've found is the two species documents, which (again) (still) shouldn't go to RC until the rest of the Species Rework project is ready to go with them. If those two files disappear I expect to be able to give a shipit.

**Comment by Turtlelord26** [comment]
Strike my last

File: CharacterCreation/02_Classes.txt [hunk 1]
```diff
@@ -427,34 +428,100 @@ Skill Proficiencies:
...
+before, the hunter has used up his one Free Tracking Action this turn, and 
+tracking the soldier again costs one Action.
+	In addition, teammates now get a bonus to the range of rolls which 
+critically hit on tracked enemies that they hit in accordance to their range and 
+weapon type. This is accomplished by you calling out weak spots that you see on 
+the target and describing shooting conditions such as the bullet drop due to 
+humidity, wind speed and direction, and the coriollis effect making the bullet 
+swerve. The bonus to critically hit is listed below. Note that if the chance to 
+critically hit exceeds the chance to hit, it does not increase the chance to 
+hit; it only garuntees that a hit will be a critical hit. If the increase in
```
Comment (Turtlelord26) [inline CharacterCreation/02_Classes.txt:490]: sp: guarantees

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]
RC Version 0.1.6 Technical Review

Changes by File (and at the end, project)

** items were formerly casualties of a play->RC merge, and are pretending to be reintroduced to RC here.

## Basic Rules

-Nonfunctional prose changes, reorderings, and standardizations. ✔️
-Feat description and rules moved into this document. ✔️
-Massively increased descriptive section on Attachments and other weapon customization. ✔️
-** Added Rolling the Average definition. Nonfunctional here, functional where used. ✔️

## Combat Rules

-Medium and Heavy suppressions changed to always hit armor, preparatory for Static armor changes. Technically functional. Testable. ✔️

## Damage Types

-Nonfunctional word standardization and reordering ✔️
-Added nonconductivity to regular armors. Functional. Testable. ✔️
Director's Note: This rule should also exist in the section of Combat Rules that talks about armor, or this may turn into a rules lawyer Gotcha. I would also like to know why this was introduced - what problem it's solving or value it's adding.

## **Movement: Microgravity, New Player Walkthrough

-** Added. Functional. Testable. ✔️

## Glossary

-Base Stats -> Primary stats, and added Secondary Stats. ✔️

## Species: Human, Species: Tundari

-Added files, part of Species rework. Functional. Testable. Incomplete pending resolution of the project. ❌

## **Classes

-** Added melee weapon and skill proficiencies to classes. ✔️

## Feats

-Miss Chance modifiers become Accuracy modifiers. Nonfunctional. ✔️
Director's Note: We might want these Accuracy instances capitalized.
-Added Juggler Feat. Very throwing weapons. Functional. Testable in conjunction with new Melee weapon damage. ✔️
-Added Duck, Bob, and Weave feat. Functional. Testable. Very like. ✔️
-** Added proficiency feats for melee weapon sizes and unarmed combat feats. Functional, tetable. ✔️
-Heavy weapon expertise 36 bonus changed to conditioning on consecutive attacks instead of high APL. Functional, testable. ✔️

## Skills

-Nonfunctional reorganization. ✔️

## **Doctor Procedures

-** Added Whateverectomy. ✔️

## **Tome of Spells

-** Deleted. ✔️

## Attachments

-Some rules moved to Basic Rules. Nonfunctional. ✔️
-Added nonfunctional prose, some restyling. ✔️

## Melee & Thrown

-Replaced old Hand-to-hand system with new Unarmed Strike, which grows in power via Unarmed Combat feats.
-Added Welding Torch weapon at lv 1, borrowing all stats from the plasma torch except damage type. Moved the plasma torch to lv 5. Functional, testable. ✔️
-Rebalanced melee weapon damages. Functional. Testable.
-Added explicit damage types to melee weapons. Technically functional but we've been treating them like this the whole time. Testable even. ✔️
-Modifications to the Grabber knife effect to be more in line with the new grapple. Functional, testable. Some balance concern has been noted. ✔️
-Range adjustments to the Throwing weapons, now independent of attributes. Functional. Testable. ✔️
-Individually noted throwing weapons may be used for melee attacks at regular damage but at an accuracy penalty. Functional. Testable. ✔️

## Items

-Added fire extinguisher. Functional. Testable. ✔️

## Hunter class rework: The Dog.

-Redone class flavor text, now has a fuller and more in-aesthetic theme. Nonfunctional. ✔️
-Add Hunter gains Attack Dog pet at lv 3. Functional. Testability, specifically of the level at which the Hunter gains the pet, check. Probably via metabalance project ✔️
-Add Attack dog. Refer to attack dog document for analysis.
-Remove Proficiency with Long Rifles. Functional. Testable. ✔️
-Mark Quarry ability removed. Functional. Testable. ✔️
-Completely redone class feat selection. Quite Functional
--Tracker: Targeted gunfire vulnerability debuff. Testable. ✔️
--Spotter: Increased Action economy and greater vulnerability. I don't get the crit bonus thing going on here.
--Pack Hunter I: Teamwork buffs. Testable ✔️
--Pack Hunter II: More teamwork buffs. Testable ✔️
--Bushwhacker: Anti-melee attack and perception buff. Testable ✔️
-The Dog. It's a functional change.
--Can't use non-food items. Testable. Would seem to imply that they can't use the toy, which is sad. Please address. ❓
--Gets cover via Nyoom: testable ✔️
--Can dodge melee with Reflex. Testable ✔️
--Has stats. Is missing Awareness save but this is calculable. Nonblocker. ❓
--Has skills. Testable ✔️
--Can distract with tricks. Testable ✔️ Director's query: Is this meant to be able to be used during combat? Seems off-flavor.
--Has an attack that does damage and automatically grapples. Testable. ✔️
--Has alternate damage for the Strike grapple action. Testable ✔️
-Dog Items are functional changes.
--Toy: Harder to test (no numbers), but testable. ✔️
--Harness: Testable ✔️
--Food: Testable. ✔️

**Comment by bleehu** [comment]
Adding a clone of this where the version bump is a separate branch

**Pull request closed without merging**